src/com/untamedears/PrisonPearl/CombatTagListener.java

```diff
@@ -25,7 +27,7 @@ public void onNpcDespawn(NpcDespawnEvent event) {
         if (event.getReason() != NpcDespawnReason.DESPAWN_TIMEOUT) {
             return;
         }
-        UUID plruuid = event.getPlayerUUID();
+        UUID plruuid = NameAPI.getUUID(event.getPlayerName());
```

Comment:
I should take down the combat tag off our jenkins page as we no longer depend on our custom implementation.